app/components/views/GovernancePage/Proposals/ProposalList/index.js

```diff
@@ -7,11 +7,16 @@ import { newProposalCounts, proposals } from "connectors";
 import { createElement as h } from "react";
 
 const PageHeader = () => (
-  <div className="proposals-community-header">
-    <div className="proposals-community-header-title"><T id="proposals.community.title" m="Proposals"/></div>
-    <p className="proposals-community-header-description"><T id="proposals.community.descr" m="Voting on community proposals allows you to have a say on how the project treasury is spent.
-        Participation in voting requires (PoS) tickets. Proposal creation, discussions and other features are available at" /> <a onClick={() => shell.openExternal("https://proposals.decred.org")}>proposals.decred.org</a></p>
-
+  <div className="proposals-community-header is-row">
+    <div className="proposals-community-header-wrapper">
+      <div className="proposals-community-header-title"><T id="proposals.community.title" m="Proposals"/></div>
+      <div className="proposals-community-header-description">
+        <T id="proposals.community.descr"
+          m="Voting on community proposals allows you to have a say on how the project treasury is spent.
+          Participation in voting requires (PoS) tickets. Proposal creation, discussions and other features are available at" />
+        <a onClick={() => shell.openExternal("https://proposals.decred.org")}>proposals.decred.org</a>
```

Comment:
Use the `ExternalLink` component so that you can link to testnet/mainnet sites correctly.

Also, pass the link as a value to the previous `proposals.community.descr` and use that as template element in the string so that the space is preserved (currently the rendered text is glued to the link).

Reply:
fixed